src/toolchain.rs

```diff
@@ -251,7 +251,14 @@ impl<'a> Toolchain<'a> {
             path_entries.push(self.path.join("bin"));
         }
 
-        env_var::prepend_path("PATH", path_entries, cmd, self.cfg.process);
+        // On Windows, we append the "bin" directory to PATH.
+        // Windows loads DLLs from PATH and the "bin" directory contains DLLs
```

Comment:
This comment somewhat contradicts the one just above that says it shouldn't be needed anymore. Perhaps the one above could use an update to explain the current understanding?

Reply:
Yeah, was thinking similar things about keeping an escape hatch around in case the new version doesn't work out. Something something learning from our mistakes...